app/js/controllers.js

```diff
@@ -1797,8 +1797,31 @@ angular.module('myApp.controllers', ['myApp.i18n'])
         })
       }
       if (selectedMessageIDs.length) {
-        ErrorService.confirm({type: 'MESSAGES_DELETE', count: selectedMessageIDs.length}).then(function () {
-          AppMessagesManager.deleteMessages(selectedMessageIDs).then(function () {
+        var peerString = $scope.curDialog.peer
+        var peerID = $scope.curDialog.peerID
+        var firstChar = peerString.charAt(0)
```

Comment:
I see; thanks a lot for clarifying! It helped a lot :) I guess I will make one more helper method in `AppPeersManager` - `isUsualGroup`.